torch/_inductor/codegen/triton_utils.py

```diff
@@ -33,6 +38,13 @@ def signature_of(arg: KernelArgType, *, size_dtype: str) -> str:
                 return "fp32"
             else:
                 return new_tye
+        elif zero_dim_cpu_tensor_list and (arg.buffer in zero_dim_cpu_tensor_list):
```

Comment:
Could we just make the above call to `is_unspec_arg` return True ? and get rid of these changes?

Reply:
I found we can't merge it to `is_unspec_arg` because just like the above code, we need different kernel signature types for is_unspec_arg and zero_dim_cpu_tensor_list

Reply:
add a `is_zero_dim_cpu_tensor` in https://github.com/pytorch/pytorch/pull/132702/commits/7be90dc77882c5f9cd155f5d2e7348c64864336e